docs/section10/10-1.md

```diff
@@ -0,0 +1,65 @@
+# Airside systems – Rule 10-1  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False    
+**Rule ID:** 10-1  
+ 
+**Rule Description:** The proposed design includes humidification (which does not comply with 90.1-2019 Section 6.5.2.4) and the baseline building design has been modeled with adiabatic humidification, as expected.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_baseline_system_types() 
+3. get_list_hvac_systems_associated_with_zone()
+4. get_hvac_zone_list_w_area()
+
+**Manual Check:** For zones modeled with humidification in the baseline and proposed,check that the baseline system serving this zone is modeled with adiabatic humidification if the specified humidification system complies with 90.1 - 2019 Section 6.5.2.4, and that the baseline system serving this zone is modeled with non-adiabatic humidification if the specified system does not comply with Section 6.5.2.4. 
+ 
+**Rule Logic:**  
+**Applicability Check 1:** 
```

Comment:
@claperle i believe you've done this in Section 4 RDS, you've added a statement explaining what each chunk of code is doing. For example, before line 31, we would add 'Get all systems in PRM with humidification specified in heating system or preheat system and get corresponding list of zones served by those systems'

You're already doing this in line 45, which makes that part clear.

Reply:
Added